pkg/sentry/kubeconfig/kubeconfig.go

```diff
@@ -342,6 +343,7 @@ func GetConfigForUser(ctx context.Context, bs service.BootstrapService, aps serv
 		message, groups)
 	*/
 
+	//service.DownloadKubeconfigAuditEvent(ctx, zap.L(), username)
```

Comment:
this seems to be commented, what's the purpose of this change ?

Reply:
zap.logger needs to be added to the kubeconfig.go file. @meain  is going to help me with that. So, have just commented the call for now.

Reply:
Let's delete this completely as of now.